walk-in-interview/src/main/webapp/log-in/applicant-log-in/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
nit: I think the file path can be /log-in/applicant/index.html instead of /log-in/applicant-log-in/index.html

Reply:
same for business log in